src/Event.java

```diff
@@ -14,11 +14,21 @@
     private ArrayList<String> attendants;
 
     // constructors
-    public Event() {
-        // empty
-    }
+    public Event(Host host) {
+     }
+    public Host getHost(){
+        return host;
+    } 
+
+     //method
+    public void Alumni(Alumni alumni){
+        alumni = guestSpeaker;}
```

Comment:
the assignment on line 25 is backwards. we are assigning guestSpeaker to the incoming alumni

Reply:
ahhh! okay i see